internal/common/disasters/environmentdefinition.go

```diff
@@ -0,0 +1,71 @@
+package disasters
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+// Island captures the location of a single island
+type Island struct {
+	id   shared.ClientID
+	x, y float64
+}
+
+// ArchipelagoGeography captures the collection of island geographies including bounding region of whole archipelago
+type ArchipelagoGeography struct {
+	islands          map[shared.ClientID]Island
+	xBounds, yBounds [2]float64
+}
+
+// disasterParameters encapsulates a disaster's information - when and how it occurs. Disaster occurring is a Bernoulli random var with `p`=GlobalProb
+// SpatialPDF determines the distribution type for the XY-location of the disaster peak. MagnitudeLambda is the lambda param in an exponential distr.
```

Comment:
![image](https://user-images.githubusercontent.com/33488131/102820601-72e68a00-43cd-11eb-894d-f2c64bc51c9f.png)

thanks for commenting this!! maybe put this above `spatialPDF string`? Then intellisense will show this comment when you hover over `spatialPDF`.

Also, how many PDFs do we have? I reckon we should use an enum (iota) -- see `shared.ClientID` for an example.

Reply:
Haha yeah - an enum is def better. Was just a bit lazy to be completely honest. But will add that to ToDo